config/jobs/kubernetes/sig-azure/sig-azure-config.yaml

```diff
@@ -21,17 +41,41 @@ presubmits:
     - master
     labels:
       preset-service-account: "true"
+      preset-azure-cred: "true"
+      preset-dind-enabled: "true"
     spec:
       containers:
-      - image: gcr.io/k8s-testimages/kubekins-e2e:v20190122-6eca4133a-master
+      - image: ritazh/kubekins-e2e:v20190126-731395d-dirty-master # update this after test-infra PRs are merged 
         args:
-        - "--repo=k8s.io/kubernetes=$(PULL_REFS)"
+        - "--job=$(JOB_NAME)"
         - "--root=/go/src"
         - "--service-account=/etc/service-account/service-account.json"
+        - "--repo=k8s.io/kubernetes=$(PULL_REFS)"
         - "--upload=gs://kubernetes-jenkins/pr-logs"
-        - --scenario=execute
+        - "--timeout=460"
+        - "--scenario=kubernetes_e2e"
         - --
-        - make
-        - test
-        - "WHAT=./pkg/cloudprovider/providers/azure/"
-
+        - "--gce-ssh=" # https://github.com/kubernetes/test-infra/pull/10054
+        - "--test=true"
+        - "--up=true"
+        - "--down=true"
+        - "--deployment=acsengine"
+        - "--build=bazel" # build kubectl binary
+        - "--provider=skeleton" # noop
+        - "--ginkgo-parallel=30"
+        - "--acsengine-agentpoolcount=3"
+        - "--acsengine-admin-username=azureuser"
+        - "--acsengine-admin-password=AdminPassw0rd"
+        - "--acsengine-creds=$AZURE_CREDENTIALS" 
+        - "--acsengine-orchestratorRelease=1.12"
+        - "--acsengine-mastervmsize=Standard_DS2_v2"
+        - "--acsengine-agentvmsize=Standard_DS2_v2"
+        - "--acsengine-hyperkube=True" # build hyperkube image
+        - "--acsengine-location=centralus"
+        - "--acsengine-public-key=$AZURE_SSH_PUBLIC_KEY_FILE"
+        - "--acsengine-template-url=https://raw.githubusercontent.com/Azure/aks-engine/master/examples/kubernetes.json"
```

Comment:
My 2 cents: You should  use a template that explicitly passes osType=Linux so it is clear that this is testing on Linux nodes.

Reply:
Also in the template, you should be aware that you need to specify distro=ubuntu for linux agents so it will respect the custom hyperkube build. The reason for this is documented here: https://github.com/Azure/aks-engine/blob/dc37161ff3693a5d9debc6eba1ad5e949d632987/docs/topics/kubernetes-developers.md

Reply:
I have set `distro=ubuntu` as default in #11032 so that we can use existing templates from https://github.com/Azure/aks-engine/master/examples

Reply:
@ritazh we should change this to a file based on a specific commit sha. So that way if it changes we dont break these tests.